testutil/daemon/ops_unix.go

```diff
@@ -1,34 +0,0 @@
-// +build !windows
-
-package daemon
-
-import (
-	"path/filepath"
-	"runtime"
-
-	"github.com/docker/docker/daemon/images"
-	"github.com/docker/docker/layer"
-
-	// register graph drivers
-	_ "github.com/docker/docker/daemon/graphdriver/register"
-	"github.com/docker/docker/pkg/idtools"
-)
-
-// WithImageService sets imageService options
-func WithImageService(d *Daemon) {
```

Comment:
Perhaps this could be  `WithImageService(*images.ImageService)` to set an image-service manually, or do we not expect that to be used for other tests?

Reply:
Since this was only created for a single test that didn't need it to be on the daemon anyways I think it's okay to remove it for now; if it's needed again in the future we can reevaluate.